keyboards/hineybush/h87a/h87a.c

```diff
@@ -42,4 +42,8 @@ void led_set_kb(uint8_t usb_led) {
 	led_set_user(usb_led);
 }
 
-
+void eeconfig_init_user(void) {  // EEPROM is getting reset!
```

Comment:
As this is at the keyboard level, the kb variant should be used
```suggestion
void eeconfig_init_kb(void) {  // EEPROM is getting reset!
```